pavelib/acceptance_test.py

```diff
@@ -46,13 +46,13 @@ def test_acceptance(options):
             'red',
             'No system specified, running tests for both cms and lms.'
         )
-        print(msg)
+        print msg
     if opts['default_store'] not in ['draft', 'split']:
         msg = colorize(
             'red',
             'No modulestore specified, running tests for both draft and split.'
         )
-        print(msg)
+        print msg
```

Comment:
Frankly, I would rather see us disable the pylint warning about using print as a function. In Py3 it's a function, and it'd be nice to not have to re-add these if/when we migrate to Py3. I don't understand why this is even a pylint issue. who the hell cares! It still works as a function!